vizro-ai/src/vizro_ai/dashboard/nodes/core_builder/model.py

```diff
@@ -0,0 +1,82 @@
+"""Contains the _get_model for the Vizro AI dashboard."""
+
+# ruff: noqa: F821
+try:
+    from pydantic.v1 import BaseModel, Field, ValidationError
+except ImportError:  # pragma: no cov
+    from pydantic import BaseModel, Field, ValidationError
+from typing import Dict
+
+from langchain_core.messages import HumanMessage
+from langchain_core.prompts import ChatPromptTemplate
+
+
+class ProxyVizroBaseModel(BaseModel):
+    """Proxy model for VizroBaseModel."""
+
+    id: str = Field(
+        "",
+        description="ID to identify model. Must be unique throughout the whole dashboard."
+        "When no ID is chosen, ID will be automatically generated.",
+    )
+
+
+SINGLE_MODEL_PROMPT = ChatPromptTemplate.from_messages(
```

Comment:
Can we double check how instructor does it? I assume there was nothing comparable in pydantic?